src/rollback_to_stable/rts_btree_walk.c

```diff
@@ -68,7 +68,9 @@ __rts_btree_walk_page_skip(
                   WT_RTS_VERB_TAG_SKIP_DEL_NULL "ref=%p: deleted page walk skipped", (void *)ref);
             else {
                 __wt_verbose_multi(session, WT_VERB_RECOVERY_RTS(session),
-                  "%p: deleted page walk skipped page_del %s", (void *)ref,
+                  WT_RTS_VERB_TAG_STABLE_PG_WALK_SKIP
```

Comment:
I believe we need a unique verbose tag to identify the message. Using an existing tag may not work.
I prefer the tag name to contain `DEL` as we are dealing with the deleted pages.

Reply:
I created a new tag and used it, see d416ce9158859eb9bafab63d0a8d55fa5cee122f.